MOBILE_CLIENT/ANDROID/_MEDIUM/IMPLICIT_PENDINGINTENT/description.md

```diff
@@ -0,0 +1,23 @@
+Implicit PendingIntent is a vulnerability which may cause security threats in the form of denial-of-service, private data theft, and privilege escalation. Please review the detailed steps below to fix the issue with your apps. 
```

Comment:
Need more details on the impact of the vulnerability and the conditions to exploit it.